priv/templates/common/error.greenbar

```diff
@@ -1,23 +1,38 @@
-~attachment title="Command Error" color="#ff3333" Caller=$initiator Pipeline=$pipeline_text "Pipeline ID"=$id Started=$started~
-~if cond=$planning_failure ~
-The pipeline failed planning the invocation:
-~br~
+~if cond=$execution_failure != ""~
```

Comment:
Yeah, I agree that it's painful to read. You're exactly right that an `else` or `elseif` would have been super helpful. The other things that might be nice is a `~comment~` tag to leave some description of what the heck is going on when it gets tangled. I'll create issues for those.